### PR TITLE
[DNM] Test add file based image resolver for rekt tests

### PR DESCRIPTION
**Pull request by creydr**
Test of #122 

Do not merge :)
/hold

### CODE REVIEW

File: openshift/e2e-common.sh [hunk 1]
```diff
@@ -96,22 +96,21 @@ function install_serverless(){
...
-  echo "Replacing knative-eventing-test-event-flaker image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-flaker|${KNATIVE_EVENTING_TEST_EVENT_FLAKER}|g" "$(dirname "$0")/../test/rekt/resources/flaker/flaker.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-heartbeats|${KNATIVE_EVENTING_TEST_HEARTBEATS}|g" "${images_file}"
 
   echo "Replacing knative-eventing-test-eventshub image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-eventshub|${KNATIVE_EVENTING_TEST_EVENTSHUB}|g" "$(dirname "$0")/../vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-eventshub|${KNATIVE_EVENTING_TEST_EVENTSHUB}|g" "${images_file}"
 
-  go_test_e2e -timeout=1h -parallel=20 ./test/rekt || failed=$?
+  go_test_e2e -timeout=1h -parallel=20 ./test/rekt --images.producer.file="${images_file}" || failed=$?
```
Comment (pierDipi) [inline openshift/e2e-common.sh:113]:
```
panic: failed to read file openshift/images.yaml: open openshift/images.yaml: no such file or directory
```

I guess an absolute file path would solve it

**Comment by creydr** [comment]
images.yaml updated with CI images. e.g.:
```
cat ./openshift/images.yaml 
knative.dev/eventing/test/test_images/print: registry.build01.ci.openshift.org/ci-op-vt0xlhjg/pipeline@sha256:552839598422a208e147214b9d9e6b48d437047589e68ab722c28becd877efbd
knative.dev/eventing/test/test_images/heartbeats: registry.build01.ci.openshift.org/ci-op-vt0xlhjg/pipeline@sha256:76114dee06b86cab8107b917eaab2ddc59af74def9f2bb4777816cffefe18ec6
knative.dev/reconciler-test/cmd/eventshub: registry.build01.ci.openshift.org/ci-op-vt0xlhjg/pipeline@sha256:bec5fb0aa61761b8e67086efc80aac8aa8fd347aaef0517618de0257cff2ebff
```

**Pull request closed without merging**